Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -18,14 +18,7 @@ extension PathHierarchy {
     /// - Returns: Returns the unique identifier for the found match or raises an error if no match can be found.
     /// - Throws: Raises a ``PathHierarchy/Error`` if no match can be found.
```

Comment:
Before, it was very easy to state that this method returned a `PathHierarchy/Error`. However, now that this method calls another very extensive method with its own inner method calls, it's more difficult to assure that this `- Throws: Raises a PathHierarchy/Error if no match can be found.` is always true.

I checked all the other throw statements inside `findNode`, and it seems like they all throw a `PathHierarchy/Error`. However, since there are multiple nested calls, I find it very easy to mistakenly throw another type of error without noticing this comment.

Any thoughts about this?

Reply:
The comment still holds true. All errors raised from `PathHierarchy/find(path:parent:onlyFindSymbols:)` are `PathHierarchy/Error`. In the future when we eventually update to Swift 6 we can adopt typed throws here to have the compiler enforce this.